azure-pipelines.yml

```diff
@@ -172,7 +172,15 @@ stages:
           - publish: $(Build.ArtifactStagingDirectory)/DotNet.Status.Web/package
             artifact: DotNetStatus
             displayName: Publish DotNet.Status.Web
-          
+
+          - script: $(Build.SourcesDirectory)\.dotnet\dotnet publish -o $(Build.ArtifactStagingDirectory)\DarcBot
+            workingDirectory: src/GitHubApps/src/DarcBot
```

Comment:
Can we remove the second "src" directory? It's a little weird to have "src/Thing/src/Stuff".  Seems like "src/Thing/Stuff" should be good. It's what all the other code in this repository looks like, anyway.

Reply:
We should also add this project to the solution, so that the "Build" phase catches it (and people can see it when they open the solution).

We almost deleted some methods this thing needed without realizing it, because it wasn't in the solution file. :-)